scripts/error-extractor/index.js

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
Can we call this `scripts/error-codes/extract-errors.js`? Then we can put code related to substituting the error codes here too later. And we can leave the codes in `scripts/error-codes/codes.json` or something.